test/vtl.test.ts

```diff
@@ -160,6 +160,19 @@ test("if statement", () => {
   );
 });
 
+test("blockless if", () => {
+  appsyncTestCase<{ num: number }, void>(
+    reflect(async ($context) => {
+      if (1 === $context.arguments.num) return;
+      if (1 === $context.arguments.num) {
+      } else return;
+    })
+  );
+
+  // https://github.com/aws-amplify/amplify-category-api/issues/592
+  // testAppsyncVelocity(templates[1]);
+});
```

Comment:
If I recall, compile.ts rewrites all blockless ifs to use blocks.

Reply:
It did not (thus the filed issue: https://github.com/functionless/functionless/issues/265) It does rewrite blockless functions to add a block with return. If we want to be 1:1, should we avoid rewriting?

Reply:
Avoiding rewrite would be good